back/node_watcher/README.md

```diff
@@ -71,11 +71,10 @@ You can also deploy a new prisma schema by setting the PRISMA_ENDPOINT environme
 1. Make sure you're logged into gcloud with an account that has access to Container Registry. You can also push to Docker Hub but for the sake of having everything in one place, we'll keep it all in GCR.
 ![gcloud auth list](images/1.gcloudauthlist.png)
 2. Run `gcloud auth configure-docker`
-3. Check the last build image version
```

Comment:
This only works if you built the latest one.
What I've put as alternative isn't working though `gcloud container images list --repository eu.gcr.io/test-installations-222013/nomidot_watcher` was outputing "Listed 0 items."

I actually tried them all from the list:
>gcloud container images list --repository eu.gcr.io/test-installations-222013                       
NAME
eu.gcr.io/test-installations-222013/nomidot-frontend
eu.gcr.io/test-installations-222013/nomidot-server
eu.gcr.io/test-installations-222013/nomidot_server
eu.gcr.io/test-installations-222013/nomidot_watcher
eu.gcr.io/test-installations-222013/nomidotwatcher

Each one was empty :interrobang:

Reply:
you should specifiy the tag

Reply:
Nice, thanks, found it with `gcloud container images list-tags eu.gcr.io/test-installations-222013/nomidot_watcher`